contracts/BucketAuction.sol

```diff
@@ -80,21 +97,30 @@ contract BucketAuction is IBucketAuction, ERC721M {
     }
 
     /**
-     * @notice begin the auction.
+     * @notice set the start and end times in unix seconds for the bucket auction.
      * @dev cannot be reactivated after price has been set.
-     * @param b set 'true' to start auction, set 'false' to stop auction.
+     * @param startTime set to unix timestamp for the auction start time.
+     * @param endTime set to unix timestamp for the auction end time.
      */
-    function setAuctionActive(bool b) external onlyOwner cannotMint {
+    function setStartAndEndTimeUnixSeconds(uint256 startTime, uint256 endTime) external onlyOwner {
```

Comment:
might also want to check that these times don't overlap with any of the existing stages, and maybe enforce some time gap between stages (~5 mins)